app/ui-react/packages/ui/src/Shared/TextEditor/velocity-lint.ts

```diff
@@ -1,12 +1,15 @@
-import { CodeMirror, TemplateType } from '@syndesis/ui';
-import { Velocity } from '@syndesis/utils';
-import i18n from '../../../../../../i18n';
+import * as CodeMirror from 'codemirror';
+import * as Velocity from 'velocityjs';
 import { AbstractLanguageLint } from './abstract-language-lint';
 import { TemplateSymbol } from './template-symbol';
 
+import 'codemirror/mode/velocity/velocity.js';
+
+export type LintVelocity = 'velocity';
+
 export class VelocityLint extends AbstractLanguageLint {
   constructor() {
-    super(TemplateType.Velocity);
+    super('velocity');
```

Comment:
`LintVelocity` here?

Reply:
Sorry, missed that it was a type, why not use a const or enum here?

Reply:
can't, it's a type :(